local/index.html

```diff
@@ -1,81 +1,17 @@
 <html>
     <head>
         <meta name="viewport" content="width=device-width,initial-scale=1">
+        <meta charset="utf-8">
         <link rel="stylesheet" href="https://unpkg.com/tachyons@4.11.1/css/tachyons.min.css"/>
         <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
         <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
         <script crossorigin src="https://unpkg.com/aphrodite@2.2.1/dist/aphrodite.umd.min.js"></script>
         <script src="https://cdn.jsdelivr.net/algoliasearch/3/algoliasearchLite.min.js"></script>
         <script src="https://sdk.v2-prod.volusion.com/element-sdk.umd.js"></script>
         <script src="component.umd.js"></script>
+        <script src="./index.js"></script>
     </head>
     <body>
         <div id="root"></div>
     </body>
-    <script>
-        var blockConfig = {
-            typography: {
-                baseFontSize: "16px",
-                fontFamily: "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'",
-                headingWeight: 700
-            },
-            color: {
-                background: "#fff",
-                link: "#777",
-                primary: '#333',
-                text: '#333'
-            },
-            globalComponents: {
-                'volComponentButton': {
-                    buttonStyles: {
-                        textColor: '#343538',
-                        textTransform: 'uppercase',
-                        backgroundColor: '#fff',
-                        borderColor: '#343538',
-                        hoverTextColor: '#fff',
-                        hoverBackgroundColor: '#343538',
-                        hoverBorderColor: '#343538',
-                        borderThickness: 'basic', // basic, thin, thick, none
-                        size: 'medium', // small, medium, large, 'block'
-                        rounded: 'small', // none, small, medium, large, pill
-                        growOnHover: false
-                    }
-                }
-            },
-            props: {
-                queryParams: {},
-                color: {
-                    background: 'yellow'
-                },
-                text: "Hello from props"
-            }
-        }
-
-        window.ElementSdk.client.configure({
-            tenant: "5b328aba89ed0900015acbcc"
-        });
```

Comment:
Where is the tenant being configured in the test block? What tenant will it use?

Reply:
@cameron-jones-volusion, I remember we talked about this briefly last week, but I don't remember what we decided on.

Reply:
I'm undecided. On the one hand, this code serves no purpose in the base starter block. On the other hand, it's a useful thing to have configured if you _do_ want to create a block that uses it.

If we do keep it in, we either need to set it to a placeholder string, like `$YOUR_TENANT_ID`, or choose an demo store to pull data from by default.

Reply:
I think a placeholder string is fine. The starter block won't use it by default anyway. I would just recommend that it's set to a variable which is located near the top of index.js